docker/ansible-in-docker/build.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -vx
+
+function buildAnsible {
+    dockerTag=oso-ansible:${ansibleVersion}
```

Comment:
This should include the distro it's built for as well (like our other containers).

e.g. `oso-rhel7-ansible` and `oso-centos7-ansible`